src/components/cards/CardItem.vue

```diff
@@ -213,8 +241,18 @@ export default {
 
 			this.$root.$emit('open-card', this.card.id)
 		},
+		triggerEditTitle() {
+			this.editingTitle = TITLE_EDITING_STATE.PENDING
+			this.$nextTick(() => {
+				this.$refs.titleContentEditable.focus()
```

Comment:
The title field was not focused when testing the pr.
this.$nextTick callbacks are executed after the vDom updates, but .focus() requires the dom to be updated already, setting the timeout with 0 allows the browser to update the DOM and the .focus() function to have an effect.

```suggestion
		setTimeout(() => {
				this.$refs.titleContentEditable.focus()
		), 0})
```

Reply:
Thanks, fixed that and also a small issue where the shortcut lock was not triggered to early so moving the mouse could also have lead to the focus not working.